<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DDA</class>
 <widget class="QMainWindow" name="DDA">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1272</width>
    <height>876</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>1272</width>
    <height>876</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>DDA</string>
  </property>
  <property name="autoFillBackground">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="zoomOut">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>20</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Zoom Out</string>
    </property>
   </widget>
   <widget class="QPushButton" name="zoomIn">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>60</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Zoom In</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>100</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Move</string>
    </property>
   </widget>
   <widget class="QPushButton" name="bird_button">
    <property name="geometry">
     <rect>
      <x>900</x>
      <y>90</y>
      <width>80</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">qproperty-icon: url(&quot;bird.png&quot;);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../../../propic.png</normaloff>../../../propic.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>160</width>
      <height>160</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="tree_button">
    <property name="geometry">
     <rect>
      <x>1160</x>
      <y>90</y>
      <width>80</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="bird.qrc">
      <normaloff>:/images/bird.png</normaloff>:/images/bird.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>100</width>
      <height>100</height>
     </size>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>900</x>
      <y>190</y>
      <width>351</width>
      <height>641</height>
     </rect>
    </property>
    <property name="title">
     <string>Settings</string>
    </property>
    <widget class="QLineEdit" name="birdSize">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>70</y>
       <width>113</width>
       <height>24</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>50</y>
       <width>58</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Size</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="birdTilt">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>160</y>
       <width>113</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>140</y>
       <width>58</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Tilt</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>220</y>
       <width>58</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Head</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="birdHead">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>240</y>
       <width>113</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>310</y>
       <width>58</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Speed</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="birdSpeed">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>330</y>
       <width>113</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>400</y>
       <width>58</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Flock</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="birdFlock">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>420</y>
       <width>113</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>500</y>
       <width>58</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Wing</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="birdWing">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>520</y>
       <width>113</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>570</y>
       <width>58</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Tail</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="birdTail">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>590</y>
       <width>113</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1272</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="bird.qrc"/>
 </resources>
 <connections/>
</ui>
